components/gantry/singleaxis/singleaxis.go

```diff
@@ -380,6 +386,11 @@ func (g *singleAxis) gantryToMotorPosition(positions float64) float64 {
 	return x
 }
 
+func (g *singleAxis) gantryToMotorSpeeds(speeds float64) float64 {
+	r := (speeds / g.mmPerRevolution) * 60
```

Comment:
we can remove gantry speeds now, and make this required and the default speed. Let's test!

Reply:
remove `gantry_mm_per_sec` from the config?

Reply:
wait actually just realized this implementation assumes we have a gantry rpm but that's not in the required config anymore

Reply:
So I think we can do `mm_per_rev` or `rpm` and work the math out for whichever one is more user friendly. I don't even know if it needs to be required because right now if there isn't `rpm` in the config we just default to 100. is that fine?

Reply:
Defaults that end up sending the motor an `rpm` of 100 if neither is specified, with a warning in the creator and move to position, are fine.